tests/components/zha/test_sensor.py

```diff
@@ -1187,6 +1192,83 @@ async def test_elec_measurement_skip_unsupported_attribute(
     assert read_attrs == supported_attributes
 
 
+class OppleCluster(CustomCluster, ManufacturerSpecificCluster):
+    """Aqara manufacturer specific cluster."""
+
+    cluster_id = 0xFCC0
+    ep_attribute = "opple_cluster"
+    attributes = {
+        0x010C: ("last_feeding_size", t.uint16_t, True),
+    }
+
+    def __init__(self, *args, **kwargs) -> None:
+        """Initialize."""
+        super().__init__(*args, **kwargs)
+        # populate cache to create config entity
+        self._attr_cache.update(
+            {
+                0x010C: 10,
+            }
+        )
```

Comment:
I'd put this in one line here:
```suggestion
        self._attr_cache.update({0x010C: 10})
```